scripts/cli_options.py

```diff
@@ -1,13 +1,203 @@
+        help="Maximum relative coinjoin fee, in fractions of the coinjoin "
+             "value, to pay to a single market maker for a transaction. Both "
+             "the limits given in --max-cj-fee-abs and --max-cj-fee-rel must "
+             "be exceeded in order to not consider a certain offer.\n"
+             "Example: 0.001 for a maximum fee of 0.1% of the cj amount")
+    parser.add_option(
+        '--order-choose-algorithm',
+        action='callback',
+        type=str,
+        default=jmclient.support.random_under_max_order_choose,
+        callback=get_order_choose_algorithm,
+        help="Set the algorithm to use for selecting orders from the order book.\n"
+             "Default: {}\n"
+             "Available options: {}"
+             .format('random_under_max_order_choose',
+                     ', '.join(order_choose_algorithms.keys())),
+        dest='order_choose_fn')
+    add_order_choose_short_options(parser)
+
+
+def add_order_choose_short_options(parser):
+    for name in sorted(order_choose_algorithms.keys()):
+        option = order_choose_algorithms[name]
+        parser.add_option(
+            option,
+            help="alias for --order-choose-algorithm={}".format(name),
+            nargs=0,
+            action='callback',
+            callback=get_order_choose_algorithm,
+            callback_kwargs={'value_kw': name},
+            dest='order_choose_fn')
+
+
+def get_order_choose_algorithm(option, opt_str, value, parser, value_kw=None):
+    value = value_kw or value
+    if value not in order_choose_algorithms:
+        raise OptionValueError("{} must be one of {}".format(
+                opt_str, list(order_choose_algorithms.keys())))
+    fn = getattr(jmclient.support, value, None)
+    if not fn:
+        raise OptionValueError("internal error: '{}' order choose algorithm not"
+                               " found".format(value))
+    setattr(parser.values, option.dest, fn)
+
+
+def get_max_cj_fee_values(config, parser_options):
+    CONFIG_SECTION = 'POLICY'
+    CONFIG_OPTION = 'max_cj_fee_'
+    # rel, abs
+    fee_values = [None, None]
+    fee_types = [float, int]
+
+    for i, option in enumerate(('rel', 'abs')):
+        value = getattr(parser_options, CONFIG_OPTION + option, None)
+        if value is not None:
+            fee_values[i] = fee_types[i](value)
+            continue
+
+        try:
+            fee_values[i] = config.get(CONFIG_SECTION, CONFIG_OPTION + option)
+        except NoOptionError:
+            pass
+
+    if len(filter(lambda x: x is None, fee_values)):
+        fee_values = prompt_user_for_cj_fee(*fee_values)
+
+    return tuple(map(lambda j: fee_types[j](fee_values[j]),
+                     range(len(fee_values))))
+
+
+def prompt_user_for_cj_fee(rel_val, abs_val):
+    msg = """Joinmarket will choose market makers randomly as long as their
+fees are below a certain maximum value, or percentage. The suggested maximums are:
+
+X = {rel_val:%}
```

Comment:
This is `rel_val` displayed as a percentage. Also, the paragraph above (line 125) refers to percentage. However the `X` value entered is a decimal fraction; this confused me when I use it, I'm quite sure others will be confused too! I'd suggest something like: `X = 0.001 (which is 0.1%)` and modify line 125 to read `or a certain fraction`.